ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -217,6 +237,35 @@ public void setV1UiCustomization(@Nullable ThreeDSecureV1UiCustomization v1UiCus
         this.v1UiCustomization = v1UiCustomization;
     }
 
+    /**
+     * Optional. Sets all UI types that the device supports for displaying specific challenge user interfaces in the 3D Secure challenge.
+     * Possible Values:
+     * 01 BOTH
+     * 02 Native
+     * 03 HTML
+     *
+     * Defaults to BOTH
+     *
+     * @param uiType {@link ThreeDSecureUiType} The UI type to request.
+     */
+    public void setUiType(@Nullable @ThreeDSecureUiType int uiType) {
```

Comment:
```suggestion
    public void setUiType(@ThreeDSecureUiType int uiType) {
```
Got a couple nits coming but not too many. This is 🔥